src/geff/interops/ctc.py

```diff
@@ -0,0 +1,217 @@
+import shutil
+from pathlib import Path
+
+import numpy as np
+import tifffile
+import typer
+import zarr
+from skimage.measure import regionprops
+from zarr.storage import StoreLike
+
+import geff
+from geff.metadata_schema import Axis, GeffMetadata
+from geff.write_arrays import write_arrays
+
+
+def ctc_tiffs_to_zarr(
+    ctc_path: Path,
+    output_store: StoreLike,
+    ctzyx: bool = False,
+    overwrite: bool = False,
+) -> None:
+    """
+    Convert a CTC file to a Zarr file.
+
+    Args:
+        ctc_path: The path to the CTC file.
+        output_store: The path to the Zarr file.
+        ctzyx: Expand data to make it (T, C, Z, Y, X) otherwise it's (T,) + Frame shape.
+        overwrite: Whether to overwrite the Zarr file if it already exists.
+    """
+    from dask.array.image import imread
+
+    array = imread(str(ctc_path / "*.tif"))
+    if ctzyx:
+        n_missing_dims = 5 - array.ndim  # (T, C, Z, Y, X)
+        expand_dims = (slice(None),) + (np.newaxis,) * n_missing_dims
+        array = array[expand_dims]
+
+    array.to_zarr(url=output_store, overwrite=overwrite)
+
+
+def from_ctc_to_geff(
+    ctc_path: Path,
+    geff_path: Path,
+    segmentation_store: StoreLike | None = None,
+    tczyx: bool = False,
+    overwrite: bool = False,
+) -> None:
+    """
+    Convert a CTC file to a GEFF file.
+
+    Args:
+        ctc_path: The path to the CTC file.
+        geff_path: The path to the GEFF file.
+        segmentation_store: The path or store to save the segmentation to.
```

Comment:
Should have option to convert image in this main function as well - you can just add path_to_image_dir and image_store optional arguments.

Reply:
done and tested